### PR TITLE
Update the c2c source doc

### PR DESCRIPTION
**Pull request by yuting-liu**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 1]
```diff
@@ -135,6 +135,27 @@ When Sumo Logic detects an issue it is tracked by [Health Events](/docs/manage/H
...
+To restart your source in the Sumo Logic platform:
+
+1. Open the Collection page, go to Manage Data > Collection > Collection.
+2. Find the source and click the information icon on the right of the row.
+3. The API usage information popup is displayed. Click the Restart Source button on the bottom left. <br>![restart source button.png](/img/collector/restart-source-button.png)</br>
+4. Click **Confirm** to send the restart request. <br>![restart source confirm.png](/img/collector/restart-source-confirm.png)</br>
+5. The bottom left of the platform will provide a notification informing you the request was successful.<br/>   ![source restart initiated.png](/img/collector/source-restart-initiated.png)
+
+#### API
+To restart your source via Sumo Management API
```
Comment (stacykor) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:151]:
```suggestion
To restart your source using the Sumo Management API
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 2]
```diff
@@ -135,6 +135,27 @@ When Sumo Logic detects an issue it is tracked by [Health Events](/docs/manage/H
   </tr>
 </table>
 
+### Restart Sources
+Sources to can be restarted when they are in ThirdPartyConfig errors.
+
+#### UI
+To restart your source in the Sumo Logic platform:
+
+1. Open the Collection page, go to Manage Data > Collection > Collection.
```
Comment (stacykor) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:144]:
```suggestion
1. Open the Collection page, go to **Manage Data > Collection > Collection**.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 3]
```diff
@@ -135,6 +135,27 @@ When Sumo Logic detects an issue it is tracked by [Health Events](/docs/manage/H
...
+2. Find the source and click the information icon on the right of the row.
+3. The API usage information popup is displayed. Click the Restart Source button on the bottom left. <br>![restart source button.png](/img/collector/restart-source-button.png)</br>
```
Comment (stacykor) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:146]:
```suggestion
3. The API usage information popup is displayed. Click the **Restart Source** button on the bottom left. <br>![restart source button.png](/img/collector/restart-source-button.png)</br>
```

**Review by stacykor — CHANGES_REQUESTED** [review]

**Comment by kimsauce** [comment]
Hi @yuting-liu , thank you for your submission! Can you please sign our CLA (linked in comment above)?

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 4]
```diff
@@ -135,6 +135,27 @@ When Sumo Logic detects an issue it is tracked by [Health Events](/docs/manage/H
...
 </table>
 
+### Restart Sources
+Sources to can be restarted when they are in ThirdPartyConfig errors.
+
+#### UI
+To restart your source in the Sumo Logic platform:
+
+1. Open the Collection page, and go to **Manage Data** > **Collection** > **Collection**.
+2. Find the source and click the information icon on the right of the row.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:145]:
```suggestion
2. Select the source and click the **information** icon on the right side of the row.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 5]
```diff
@@ -135,6 +135,27 @@ When Sumo Logic detects an issue it is tracked by [Health Events](/docs/manage/H
   </tr>
 </table>
 
+### Restart Sources
+Sources to can be restarted when they are in ThirdPartyConfig errors.
+
+#### UI
+To restart your source in the Sumo Logic platform:
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:142]:
```suggestion
To restart your source in the Sumo Logic platform, follow the steps below:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 6]
```diff
@@ -135,6 +135,27 @@ When Sumo Logic detects an issue it is tracked by [Health Events](/docs/manage/H
...
+To restart your source in the Sumo Logic platform, follow the steps below:
+
+1. Open the Collection page, and go to **Manage Data** > **Collection** > **Collection**.
+2. Select the source and click the **information** icon on the right side of the row.
+3. The API usage information popup is displayed. Click the **Restart Source** button on the bottom left. <br>![restart source button.png](/img/collector/restart-source-button.png)</br>
+4. Click **Confirm** to send the restart request. <br>![restart source confirm.png](/img/collector/restart-source-confirm.png)</br>
+5. The bottom left of the platform will provide a notification informing you the request was successful.<br/>   ![source restart initiated.png](/img/collector/source-restart-initiated.png)
+
+#### API
+To restart your source using the Sumo Management API
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:151]:
```suggestion
To restart your source using the Sumo Management API, follow the method below:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 7]
```diff
@@ -135,6 +135,27 @@ When Sumo Logic detects an issue it is tracked by [Health Events](/docs/manage/H
   </tr>
 </table>
 
+### Restart Sources
+Sources to can be restarted when they are in ThirdPartyConfig errors.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:139]:
```suggestion
You can restart the Sources when they are in ThirdPartyConfig errors.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 8]
```diff
@@ -135,6 +135,27 @@ When Sumo Logic detects an issue it is tracked by [Health Events](/docs/manage/H
   </tr>
 </table>
 
+### Restart Sources
+You can restart the Sources when they are in ThirdPartyConfig errors.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:139]:
```suggestion
When Sources encounter ThirdPartyConfig errors, they can now be restarted. Both the Sumo Logic UI and Sumo Logic API allow you to restart the Sources.
```

**Review by JainM6 — APPROVED** [review]

**Pull request merged**